dffml/high_level.py

```diff
@@ -258,3 +258,60 @@ async def predict(
                 )
                 if update:
                     await sctx.update(record)
+
+
+async def load(source: BaseSource, *args: str) -> AsyncIterator[Record]:
+    """
+    Returns records from a source.
+
+    For each record given, call
+    :py:func:`update <dffml.source.source.BaseSourceContext.update>` on the
+    source. Effectively saving all the records to the source.
+
+    Parameters
+    ----------
+    source : BaseSource
+        Data source to use. See :doc:`/plugins/dffml_source` for sources and
+        options.
+    *args : str
+        Records to be returned. If None all the records in a source will be returned.
```

Comment:
`args` will never be `None`, just an empty list.

```suggestion
        Records to be returned. If empty, all the records in a source will be returned.
```